lunamark/reader/markdown.lua

```diff
@@ -1325,27 +1319,28 @@ function M.new(writer, options)
                                                      writer.string(infostring))
                          end
 
+  local function increment_div_level(increment)
+    local function update_div_level(s, i, current_level) -- luacheck: ignore s i
+      current_level = tonumber(current_level)
+      local next_level = tostring(current_level + increment)
+      return true, next_level
+    end
 
-  larsers.FencedDivBegin = larsers.fenced_div_begin
-                         * Cmt(P(true), function(_, i)
-                             -- keep track how deep nested we are
-                             div_level = div_level + 1
-                             return i
-                           end)
-
-  larsers.FencedDivEnd   = larsers.fenced_div_end
-                         * Cmt(P(true), function(_, i)
-                             local result
-                             -- check whether we are at top level and decrement
-                             -- div_level otherwise
-                             if div_level <= 0 then
-                               result = nil
-                             else
-                               div_level = div_level - 1
-                               result = i
-                             end
-                             return result
-                           end)
+    return Cg( Cmt(Cb("div_level"), update_div_level)
+             , "div_level")
+  end
+
+  larsers.FencedDiv = larsers.fenced_div_begin * increment_div_level(1)
+                    * parsers.blanklines
```

Comment:
```suggestion
                    * parsers.skipblanklines
```

Seems this solves the problem with
```
::: { .cit }

I am a _fenced_ div
:::

Rendering as an empty div

Reply:
Moving `parsers.blanklines` in the Ct capture would work too, but the output is less convincing (we get a blank line).